roles/ironic-common/templates/etc/ironic/ironic.conf

```diff
@@ -50,10 +56,38 @@ rabbit_password = {{ secrets.rabbit_password }}
 
 [glance]
 glance_host={{ endpoints.main }}
+auth_strategy=keystone
```

Comment:
So a bunch of this file uses <key> = <value> but this block is <key>=<value>.

Any way we could be consistent throughout the file?